drivers/linux/chipsec_km.c

```diff
@@ -544,6 +544,13 @@ int __weak phys_mem_access_prot_allowed(struct file *file,
         return 1;
 }
 
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(4,15,4)
```

Comment:
Do we need to worry about the case where ARCH_HAS_VALID_PHYS_ADDR_RANGE is not defined?  If so the function will be defined twice.

Reply:
Agreed.  Would #undef ARCH_HAS_VALID_PHYS_ADDR_RANGE be a better solution?

Reply:
Yes, update the change to check the kernel version and that ARCH_HAS_VALID_PHYS_ADDR_RANGE is defined before adding your function definition.

Reply:
@BrentHoltsclaw can you do some testing on a few different distros to make sure this does not break anything.  If it looks good we can go with this solution for now and fix any corner cases if they show up.